scripts/Dockerfile

```diff
@@ -12,7 +12,21 @@ ENV PYTHON_VERSION=${PYTHON_VERSION}
 ENV PYSTAN_VERSION=${PYSTAN_VERSION}
 ENV DOCKER_BUILD=true
 
-RUN apt-get update && apt-get install -y git build-essential \
+# For Sphinx documentation builds
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+ENV TRAVIS_BRANCH=${TRAVIS_BRANCH}
+ENV TRAVIS_PULL_REQUEST=${TRAVIS_PULL_REQUEST}
+ENV TRAVIS_REPO_SLUG=${TRAVIS_REPO_SLUG}
+ENV TRAVIS_TAG=${TRAVIS_TAG}
+ENV GH_TOKEN=${GH_TOKEN}
+ENV GH_REPO_SLUG=${GH_REPO_SLUG}
```

Comment:
we need at least TRAVIS_JOB_ID

https://docs.travis-ci.com/user/coveralls/#using-coveralls-with-docker-builds

Reply:
This link is very helpful!